frodo.js

```diff
@@ -1,126 +1,128 @@
-const largestNumber = Math.max(...numbers);
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.